packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -131,8 +132,8 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   /// Makes the magnifier lens look so that the colors are normal through
   /// the lens and partially grayed out around it.
   Widget _buildMagnifierScreen() {
-    final Color foreground = widget.backgroundColor?.withAlpha(
-      (widget.backgroundColor?.alpha * _kForegroundScreenOpacityFraction).toInt()
+    final Color foreground = widget.backgroundColor.withAlpha(
+      (widget.backgroundColor.alpha * _kForegroundScreenOpacityFraction).toInt()
```

Comment:
the inner ? seems bogus, but I think the outer one is intended. Pity about the `toInt` though...
cc @xster

Reply:
I guess we need better testing here.

Reply:
Ah, the middle line is definitely bogus. My dart-fu isn't strong enough to understand why it still returns null to foreground currently. Will add a test.